### PR TITLE
Convert to using NuGet packages and removing all static dependencies on Gateway SDK

### PR DESCRIPTION
**Pull request by barnstee**
Also cleaned up project and documentation.
Also fixed endpoint URL string comparison bug.

### CODE REVIEW

File: Opc.Ua.Client.Module.csproj [hunk 1]
```diff
@@ -30,30 +30,48 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\x86\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
```
Comment (marcschier) [inline Opc.Ua.Client.Module.csproj:36]: Why didn't we need to compile x86 before?  And why now?

**Review by marcschier — APPROVED** [review]

**Review by mregen — APPROVED** [review]
Sweet!

File: Opc.Ua.Client.Module.csproj (hunk 1 above)
Comment (barnstee) [reply Opc.Ua.Client.Module.csproj:36]: https://social.msdn.microsoft.com/Forums/vstudio/en-US/df468308-883f-41d0-b66c-901df435e98d/can-you-pinvoke-a-32bit-dll-from-64net-assembly?forum=csharpgeneral
We should habe compiled for x86 all along. So far we were just lucky that no one created a 64 bit entry point.

Comment (barnstee) [reply Opc.Ua.Client.Module.csproj:36]: https://social.msdn.microsoft.com/Forums/vstudio/en-US/df468308-883f-41d0-b66c-901df435e98d/can-you-pinvoke-a-32bit-dll-from-64net-assembly?forum=csharpgeneral
We should habe compiled for x86 all along. So far we were just lucky that no one created a 64 bit entry point.

Comment (barnstee) [reply Opc.Ua.Client.Module.csproj:36]: https://social.msdn.microsoft.com/Forums/vstudio/en-US/df468308-883f-41d0-b66c-901df435e98d/can-you-pinvoke-a-32bit-dll-from-64net-assembly?forum=csharpgeneral
We should habe compiled for x86 all along. So far we were just lucky that no one created a 64 bit entry point.

**Pull request merged**